src/html/partials/key-factors.html

```diff
@@ -3,37 +3,46 @@
   <h2 class="sm-title">Why People Don't Adhere</h2>
   <p class="lg-text">A myriad of factors conspire to prevent people from sticking to their treatment programs. But a few are most telling.</p>
 
-  <!-- <div class="factors-section__human-figure-wrapper">
-    <div class="factors-section__human-figure-gear"></div>
-    <div class="factors-section__human-figure-heart"></div>
-    <div class="factors-section__human-figure-dollar"></div>
-    <div class="factors-section__human-figure-tally"></div>
+  <!-- <div class="factors-section__human-figure">
+    <div class="factors-section__human-figure__gear"></div>
+    <div class="factors-section__human-figure__heart"></div>
+    <div class="factors-section__human-figure__dollar"></div>
+    <div class="factors-section__human-figure__tally"></div>
   </div> -->
 
+  <!-- for <h3> with orange text, I'm adding class '-orange' and deviating slightly from BEM -->
+
   <ul class="factors-section__list">
     <li class="factors-section__list__item">
```

Comment:
aha, so here we have the `factors-section__list` which has a very similar style to the `behind-scenes-section__list` would it make sense to condense these into one style name?